### ISSUE TITLE
Check dependency on PyPlot

### ISSUE THREAD

**Issue by christiankral** [christiankral/EE.jl#3]
In Julia 1.1.0 the following message is gernerated:
```
using EE
[ Info: Precompiling EE [c638b1ac-f202-5047-8f20-c6a32b8565b5]
┌ Warning: Package EE does not have PyPlot in its dependencies:
│ - If you have EE checked out for development and have
│   added PyPlot as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with EE
└ Loading PyPlot into EE from project dependency, future warnings for EE are suppressed.
```

**Comment by christiankral** [comment]
Additional message
```
/usr/local/lib/python3.6/dist-packages/matplotlib/__init__.py:846: MatplotlibDeprecationWarning: 
The text.latex.unicode rcparam was deprecated in Matplotlib 2.2 and will be removed in 3.1.
  "2.2", name=key, obj_type="rcparam", addendum=addendum)
/usr/local/lib/python3.6/dist-packages/matplotlib/__init__.py:846: MatplotlibDeprecationWarning: 
The text.latex.unicode rcparam was deprecated in Matplotlib 2.2 and will be removed in 3.1.
  "2.2", name=key, obj_type="rcparam", addendum=addendum)
ERROR: LoadError: LoadError: LoadError: UndefVarError: @doc_str not defined
Stacktrace:
 [1] top-level scope
 [2] include at ./boot.jl:326 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include at ./sysimg.jl:29 [inlined]
 [5] include(::String) at /home/christian/.julia/dev/EE/src/EE.jl:2
 [6] top-level scope at none:0
 [7] include at ./boot.jl:326 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1038
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] top-level scope at none:2
 [11] eval at ./boot.jl:328 [inlined]
 [12] eval(::Expr) at ./client.jl:404
 [13] top-level scope at ./none:3
in expression starting at /home/christian/.julia/dev/EE/src/plot.jl:36
in expression starting at /home/christian/.julia/dev/EE/src/plot.jl:36
in expression starting at /home/christian/.julia/dev/EE/src/EE.jl:10
ERROR: Failed to precompile EE [c638b1ac-f202-5047-8f20-c6a32b8565b5] to /home/christian/.julia/compiled/v1.1/EE/rU4jN.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1197
 [3] _require(::Base.PkgId) at ./loading.jl:960
 [4] require(::Base.PkgId) at ./loading.jl:858
 [5] require(::Module, ::Symbol) at ./loading.jl:853
```

**Comment by christiankral** [comment]
Fixed by 5bf8b0ff0078819624e2a18e72dc6f179730c94d
